Readme.md

```diff
@@ -148,6 +166,41 @@ The current lead maintainer is [Douglas Christopher Wilson](https://github.com/d
 
 [List of all contributors](https://github.com/expressjs/express/graphs/contributors)
 
+### Collaborators
+
+* [Jon Church](https://github.com/jonchurch) -
+  **Jon Church** <<me@jonchurch.com>>
+* [Wes Todd](https://github.com/wesleytodd) -
+  **Wes Todd** <<wes@wesleytodd.com>>
+
+### Technical Committee
+
+* [Ulises Gascon](https://github.com/UlisesGascon) -
+  **Ulises Gascon** <<ulisesgascongonzalez@gmail.com>>
+* [Jon Church](https://github.com/jonchurch) -
+  **Jon Church** <<me@jonchurch.com>>
+* [Wes Todd](https://github.com/wesleytodd) -
+  **Wes Todd** <<wes@wesleytodd.com>>
+* [Linus Unnebäck](https://github.com/LinusU) -
+  **Linus Unnebäck** <<linus@folkdatorn.se>>
+* [Blake Embrey](https://github.com/blakeembrey) -
+  **Blake Embrey** <<>>
+* [Jean Burellier](https://github.com/sheplu) -
+  **Jean Burellier** <<>>
+* [Rand McKinney](https://github.com/crandmck) -
+  **Rand McKinney** <<>>
+
```

Comment:
Can we include an `emeriti` section using a collapsible structure? This will help us to keep the history of the project and provide recognition. I will also suggest to include this section in all the groups :)